etc/supervisor/conf.d/vta.amigocloud.com.conf

```diff
@@ -0,0 +1,53 @@
+[program:modeify]
```

Comment:
:x: 
- [ ] Please fix this supervisor file, it has services and events for things that are not related to OTP, leftover crumbs of code that has been deprecated or split separately out for the modeify aka trips.vta.org project.